net/iputils/Makefile

```diff
@@ -0,0 +1,144 @@
+#
+# Copyright (C) 2006-2010 OpenWrt.org
+# Copyright (C) 2019 Noah Meyerhans <frodo@morgul.net>
+#
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=iputils
+PKG_VERSION:=20190709
+PKG_RELEASE:=1
+PKG_MAINTAINER:=Noah Meyerhans <frodo@morgul.net>
+PKG_LICENSE:=BSD-3-Clause
+PKG_LICENSE_FILES:=LICENSE
+PKG_CPE_ID:=cpe:/a:iputils_project:iputils
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/iputils/iputils/tar.gz/s$(PKG_VERSION)?
+PKG_HASH:=a15720dd741d7538dd2645f9f516d193636ae4300ff7dbc8bfca757bf166490a
+PKG_BUILD_DIR:=$(BUILD_DIR)/iputils-s$(PKG_VERSION)
+PKG_BUILD_DEPENDS:=meson/host
+include $(INCLUDE_DIR)/package.mk
+include ../../devel/meson/meson.mk
+
+MESON_ARGS:=-Dprefix=/usr \
+            -DBUILD_MANS=false \
+            -DBUILD_HTML_MANS=false \
+            -DUSE_CAP=false \
+            -DUSE_CRYPTO=kernel \
+            -DBUILD_PING=true \
+            -DBUILD_ARPING=true \
+            -DBUILD_CLOCKDIFF=true \
+            -DBUILD_TRACEPATH=true \
+            -DBUILD_TFTPD=true
+
+define Package/iputils/Default
+  SECTION:=net
+  CATEGORY:=Network
+  TITLE:=iputils
+  URL:=https://github.com/iputils/iputils
+endef
+
+### ping
+
+define Package/iputils-ping
+  $(call Package/iputils/Default)
+  TITLE:=iputils-ping
```

Comment:
So I mentioned using kernel instead of OpenSSL. However, I think you have to add a depends for knod-crypto-md5.

Reply:
Yes, it seems so. Added. The behavior if it's missing is somewhat surprising, but that's a matter I can raise with the iputils upstream.

I am somewhat skeptical that the memory overhead of having the md5 module loaded in the kernel (even after ping has exited) is really the most efficient use of resources.

Reply:
Well, on ath79, kmod-crypto-md5 is 4.1 KB. libopenssl is 967.7 KB. I don't think the latter makes too much sense.

Reply:
Also, the former I believe benefits from hardware acceleration if applicable. The latter does not by default. It must be configured.